scss/_config.scss

```diff
@@ -40,13 +40,13 @@
 //);
 
 // vertical spacing
-$vertical_spacing: (
-        "values":0 1 2 4 6 8 12 16 24 32 48 64 96 128,
-        "md": 0 1 2 4 6 8 12 16 24 30 40 50 60 70,
-);
+//$vertical_spacing: (
+//        "values":0 1 2 4 6 8 12 16 24 32 48 64 96 128,
+//        "md": 0 1 2 4 6 8 12 16 24 30 40 50 60 70,
+//);
```

Comment:
Unrelated changes.

Reply:
![image](https://github.com/viivue/atomic-css/assets/77733880/a7696357-f656-4396-8dd5-0e4999bc599d)
I don't know whether this is the change of code in the image above. 

It doesn't show any changes when I compare my local with `staging` branch in the image below
![image](https://github.com/viivue/atomic-css/assets/77733880/d4479b9b-c3ba-402e-beaa-8c9cbddecbf9)